src/codecs.ts

```diff
@@ -871,3 +873,61 @@ export const CookieCsvInput = t.intersection([
 
 /** Type override */
 export type CookieCsvInput = t.TypeOf<typeof CookieCsvInput>;
+
+/// //////////////////////////////////////
+// Transcend Proxy AI Server policies   //
+/// //////////////////////////////////////
+export const Policies = t.literal('redactEmail');
```

Comment:
I think if you were to add more policies in the future, the codec would come out differently compared to what it is right now -- we should probably start off with a singleton union here to prevent that from happening?
```suggestion
export const Policies = t.union([t.literal('redactEmail')]);
```

Reply:
I agree with you. I tried that but the io-ts complains if you have a union with a single element. I'll change this to union once we have more policies.

Reply:
We actually do have another policy called `log` so I'll udpdate this now!

Reply:
done